lib/helpers/RoutesHelper.dart

```diff
@@ -2,6 +2,8 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+import 'package:trovami/signinpage.dart';
+import 'package:trovami/ui/splash/splash_screen.dart';
```

Comment:
I was thinking of a folder that has only the UI elements. Maybe we can name it better. Not really sure what. Maybe "views"? I thought of ui because it makes it clear that it has only "UI" elements in it and should contain no business logic.
It will contain all our screens and common widgets. 

How about this, 

lib
|- views
|---- widgets (all the common widgets used across screens)
|---- screens (.dart files for each screen)